integration-tests/fhir/src/main/java/org/apache/camel/quarkus/component/fhir/it/FhirR5Resource.java

```diff
@@ -1195,4 +1198,19 @@ private IGenericClient getFhirClient() {
         FhirComponent component = context.getComponent("fhir-r5", FhirComponent.class);
         return FhirHelper.createClient(component.getConfiguration(), context);
     }
+
+    // TODO: Remove this. Work around bug in R5 link retrieval in hapi-fhir-core. Remove when upgraded to hapi-fhir-core >= 5.6.84
```

Comment:
Maybe a follow up here ?

Reply:
I can create an issue for it. The fix in fhir-core is not released yet.

Reply:
https://github.com/apache/camel-quarkus/issues/4566